xml/ay_bigfile.xml

```diff
@@ -2556,791 +2587,1071 @@ openssl x509 -noout -fingerprint -sha256
-          out before &ay; starts the partition calculation.
-         </para>
-<screen>&lt;initialize config:type="boolean"&gt;true&lt;/initialize&gt;</screen>
-        </entry>
-        <entry>
-         <para>
-          Optional. The default is <literal>false</literal>.
-         </para>
-        </entry>
-       </row>
-       <row>
-        <entry>
-         <para>
-          <literal>partitions</literal>
+          <literal>lvm</literal>
          </para>
         </entry>
         <entry>
          <para>
-          A list of &lt;partition&gt; entries (see
-          <xref linkend="ay-partition-configuration"/>).
+          Boolean
+          <screen>&lt;lvm config:type="boolean"&gt;true&lt;/lvm&gt;</screen>
          </para>
-<screen>&lt;partitions config:type="list"&gt;
-  &lt;partition&gt;...&lt;/partition&gt;
-  ...
-&lt;/partitions&gt;</screen>
         </entry>
         <entry>
          <para>
-          Optional. If no partitions are specified, &ay; will create a
-          reasonable partitioning (see
-          <xref linkend="ay-auto-partitioning"/>).
+          Create a LVM-based proposal. The default is <literal>false</literal>.
          </para>
         </entry>
        </row>
        <row>
         <entry>
          <para>
-          <literal>pesize</literal>
+          <literal>resize_windows</literal>
          </para>
         </entry>
         <entry>
          <para>
-          This value only makes sense with LVM.
+          Boolean
+          <screen>&lt;resize_windows config:type="boolean"
+&gt;false&lt;/resize_windows&gt;</screen>
          </para>
-<screen>&lt;pesize&gt;8M&lt;/pesize&gt;</screen>
         </entry>
         <entry>
          <para>
-          Optional. Default is 4M for LVM volume groups.
+          When set to <literal>true</literal>, &ay; resizes Windows partitions
+          if needed to make room for the installation.
          </para>
         </entry>
        </row>
        <row>
         <entry>
          <para>
-          <literal>use</literal>
+          <literal>windows_delete_mode</literal>
          </para>
         </entry>
         <entry>
-         <para>
-          Specifies the strategy &ay; will use to partition the hard
-          disk.
-         </para>
          <para>
           Choose between:
+          <itemizedlist mark="bullet" spacing="normal">
+           <listitem>
+            <para>
+             <literal>none</literal> does not remove Windows partitions.
+            </para>
+           </listitem>
+           <listitem>
+            <para>
+             <literal>ondemand</literal> removes Windows partitions if needed.
+            </para>
+           </listitem>
+           <listitem>
+            <para>
+             <literal>all</literal> removes all Windows partitions.
+            </para>
+           </listitem>
+          </itemizedlist>
+          <screen>&lt;windows_delete_mode config:type="symbol"
```

Comment:
Are the screens in this table supposed to be one line?

Reply:
Yes, that's the idea.